presto-raptor-legacy/src/main/java/io/prestosql/plugin/raptor/legacy/RaptorTableHandle.java

```diff
@@ -50,6 +56,8 @@ public RaptorTableHandle(
             @JsonProperty("bucketCount") OptionalInt bucketCount,
             @JsonProperty("organized") boolean organized,
             @JsonProperty("transactionId") OptionalLong transactionId,
+            @JsonProperty("constraint") TupleDomain<RaptorColumnHandle> constraint,
+            @JsonProperty("bucketAssignments") Optional<List<String>> bucketAssignments,
```

Comment:
This confusing, as it will never be present in JSON since we `@JsonIgnore` below, but also avoids duplicating a long constructor just to skip a single argument, so I think this is fine.